mod/aalborg_theme/views/default/css/elements/misc.php

```diff
@@ -62,3 +62,7 @@
 	display: none;
 	padding: 0;
 }
+
+.elgg-highlight {
```

Comment:
Too generic of a class name I think. I mean unless we literally want highlight to always change the background to this color. Couldn't there be another situation where we want to highlight something but change it in other ways?